### PR TITLE
camel: fix broken indexing introduced by camel-core split

### PR DESCRIPTION
**Pull request by lburgazzoli**

### CODE REVIEW

**Comment by stuartwdouglas** [comment]
I am not sure why CI did not run here. Can you rebase on master and force push and see if that fixes it?

**Comment by lburgazzoli** [comment]
done

File: extensions/camel/camel-core/deployment/src/main/java/io/quarkus/camel/core/deployment/CamelInitProcessor.java [hunk 1]
```diff
@@ -69,7 +69,7 @@ AdditionalBeanBuildItem camelRuntimeProducer(BuildProducer<BeanContainerListener
     }
 
     @Record(ExecutionTime.STATIC_INIT)
-    @BuildStep(applicationArchiveMarkers = CamelSupport.CAMEL_SERVICE_BASE_PATH)
+    @BuildStep(applicationArchiveMarkers = { CamelSupport.CAMEL_SERVICE_BASE_PATH, "org/apache/camel" })
```
Comment (gsmet) [inline extensions/camel/camel-core/deployment/src/main/java/io/quarkus/camel/core/deployment/CamelInitProcessor.java]: Minor gripe but should we have `CamelSupport.CAMEL_ROOT_PACKAGE`?

Comment (stuartwdouglas) [reply extensions/camel/camel-core/deployment/src/main/java/io/quarkus/camel/core/deployment/CamelInitProcessor.java]: I think that would be better, but I don't think it is worth holding up the PR over.

**Review by stuartwdouglas — APPROVED** [review]

**Pull request merged**